src/components/switch/styles/Switch.styles.ts

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { SwitchProps } from '../content/switch_types';
+import { SizeBtn, SizeSlider, position } from '../utils/constants';
+export const Wrapper = styled.div<SwitchProps>(({}) => ({
+  position: 'relative',
+}));
+
+export const Label = styled.div<SwitchProps>(({ theme, value }) => ({
+  ...theme.typography.body1,
+  ...position[value],
+  position: 'absolute',
+}));
+
+export const ToggleButton = styled.div<SwitchProps>(
+  ({ checked, size, color, theme }) => ({
+    ...SizeBtn[size],
```

Comment:
in chie ?

Reply:
ye toggle button ye dayere sefid dakheleshe in bara one